website/src/css/components/_Cookiebot.scss

```
@@ -0,0 +1,42 @@
+#CybotCookiebotDialog,
+#CybotCookiebotDialog a,
+#CybotCookiebotDialog div,
+#CybotCookiebotDialogBodyContentControls,
+#CybotCookiebotDialogBodyContentTitle,
+#CybotCookiebotDialogBodyButtons a {
+    font-family: Circular!important;
+}
+#CybotCookiebotDialog {
+    opacity: .90 !important;
+}
+#CybotCookiebotDialogBodyContent,
+#CybotCookiebotDialogBodyContentTitle {
+    font-weight:300!important;
+}
+#CybotCookiebotDialogBody {
+    max-width: 1000px!important;
+}
```

```suggestion
}
#CybotCookiebotDialogBodyContentText {
    font-size: 0.9rem !important;
}
```

The text at the current size is difficult to read.